docs/make-ckan-live.md

```diff
@@ -0,0 +1,46 @@
+# Making CKAN Live
+
+## Point your live URL to the correct DNS
+
+- Go to the AWS EC2 Dashboard.
+- Select your EC2 instance and copy the Public DNS URL.
+  ![image-20200720113944879](../assets/public_dns.png)
+- On your hosting service, create a `CNAME` record with your live site URL. Point it to the Public DNS URL you just copied.
+  - Example: `CNAME		data.openpeoria.com`
+
+
+
+## Point CKAN Config to your live URL
+
+- SSH into your CKAN EC2 instance.
+- Open the `production.ini` file and change the `ckan.site_url` variable.
+  ```bash
+  sudo vim /lib/ckan/default/src/ckan/production.ini
+  ```
+  ```ini
+  ckan.site_url = {your_live_site}
+  # example - ckan.site_url = https://data.openpeoria.com
+  ```
+
+
+
+## Set up HTTPS
+
+### Add an Acme Challenge
+
+To get the LetsEncrypt SSL certificate, you will need to create a `TXT` record with an Acme challenge.
+
+- Add a `TXT` record to your hosting service with a key of `_acme-challenge.{your_live_site}`and a value of `Op6S2FCJ720AGVJGnYg4Hs00mCiUdm-bxg9Tcsgh1zE`.
+  - *Example*: ![img](../assets/txt_record.png)
```

Comment:
Part of me makes me wonder if this is even needed, I believe it came from https://github.com/openpeoria/AWS-CKAN/issues/8, but I don't see `_acme-challenge` listed anywhere in tutorial. It also doesn't referenced in https://handbook.nerevu.com/ckan-https.html#setting-up-https-on-ckan-ec2

Reply:
I've disabled this record, so wait a few hours and see if anything breaks...

Reply:
The [original tute](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/SSL-on-amazon-linux-2.html#letsencrypt) uses `http challenge`, and the txt entry is a `dns challenge`. What type is used [now](https://handbook.nerevu.com/ckan-https.html)?

https://letsencrypt.org/docs/challenge-types/

Reply:
Ya, I was wondering the same thing. 

I don't think disabling the record will do anything to our site right now because we've already received the certificate. It looks like the TXT record is only needed when issuing a certificate (and probably updating).

A DNS Challenge is used right now, but [the article](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge) you referenced makes me think that we could try an `http challenge` if we wanted to. That would be more automated. I'm not the most knowledgeable about this kind of stuff, so I just went with the _acme-challenge and left it alone because it worked 😆 

So where should we go from here?

Reply:
Hmmm, can you link to the handbook and tute sections you followed that sets up the DNS challenge?

Reply:
just making sure you saw the above comment @sottom

Reply:
Yes! I'll get it done.

Reply:
I searched through my browsing history and couldn't find the forum that told me to try doing a TXT record, but I will add a link to [this article](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge) and explain it a little bit. Sound good?

Reply:
I added the information

Reply:
Sounds good, can you add an issue re: switching to an http challenge ?

Reply:
Sure thing!

Reply:
Added here - #17